adlfs/spec.py

```diff
@@ -374,7 +374,12 @@ def __init__(
             and self.sas_token is None
             and self.client_id is not None
         ):
-            self.credential = self._get_credential_from_service_principal()
+            (
```

Comment:
Maybe causing a linting issue: https://dev.azure.com/dask-dev/dask/_build/results?buildId=1724&view=logs&j=d090b295-8b65-5b33-df45-4b0e4ec519f8&t=34b50283-3bbf-53e1-90cc-f6f26ef783f5. Can you run `black` on your changes (or set up pre-commit to do it automatically).

Reply:
Yep - just rebased with the linting fixes.